src/sqlfluff/rules/L009.py

```diff
@@ -9,6 +9,7 @@
 @document_fix_compatible
 class Rule_L009(BaseRule):
     """Files must end with a trailing newline."""
+    _check_docstring = False
```

Comment:
Could we not add an example for this? With a symbol for newline (like we use a symbol for "space" in L001)?

Reply:
But we here talking about file, not sql syntax. So I am not sure use sql text would be a good idea.

Reply:
And maybe we should use bash here?
```bash
> cat filename_anti.sql
select
        id
from foo
>
> cat filename_best.sql
select
        id
from foo

>
```

Reply:
I add describe and make sense for users